frontend/public/components/utils/build-strategy.tsx

```diff
@@ -21,45 +22,74 @@ export const BuildStrategy: React.SFC<BuildStrategyProps> = ({ resource, childre
   const pushSecret = _.get(resource, 'spec.output.pushSecret');
   const resourceLimits = _.get(resource, 'spec.resources.limits');
   const triggers = _.map(resource.spec.triggers, 'type').join(', ');
+  const { t } = useTranslation();
 
   return (
     <dl className="co-m-pane__details">
       {children}
-      <DetailsItem label="Type" obj={resource} path="spec.strategy.type" />
-      <DetailsItem label="Git Repository" obj={resource} path="spec.source.git.uri" hideEmpty />
-      <DetailsItem label="Git Ref" obj={resource} path="spec.source.git.ref" hideEmpty />
+      <DetailsItem label={t('build-strategy~Type')} obj={resource} path="spec.strategy.type" />
+      <DetailsItem
+        label={t('build-strategy~Git Repository')}
```

Comment:
This same case change will need to be made to each of the properties below. (I haven't commented on each.)

```suggestion
        label={t('build-strategy~Git repository')}
```